arelle/plugin/validate/ESEF/Util.py

```diff
@@ -24,7 +24,7 @@
 
 # check if a modelDocument URI is an extension URI (document URI)
 # also works on a uri passed in as well as modelObject
-def isExtension(val: ValidateXbrl, modelObject: ModelObject | None) -> bool:
+def isExtension(val: ValidateXbrl, modelObject: ModelObject | str | None) -> bool:
```

Comment:
Not sure why this wasn't caught in the first PR but the line `if isinstance(modelObject, str):` requires `modelObject` to also be of type `str`.